types/react-stripe-elements/index.d.ts

```diff
@@ -59,7 +59,7 @@ export namespace ReactStripeElements {
 
 		onFocus?(event: ElementChangeResponse): void;
 
-		onReady?(): void;
+		onReady?(el: any): void;
```

Comment:
It may be worth typing this as `HTMLElement` or `HTMLInputElement`.

```suggestion
		onReady?(el: HTMLElement): void;
```

Reply:
Yeah, you are right. Fixed it. 
I've also updated accordingly test file and checked all the Contribution guidelines.

Cheers